content/posts/irods-development-update-january-2023.md

```diff
@@ -0,0 +1,240 @@
+Title: iRODS Development Update: January 2023
+Date: 2023-01-23 12:00
+Author: Kory Draughn
+Slug: irods-development-update-january-2023
+Status: published
+
+
+Happy New Year!
+
+It's 2023 and what better way to kick-start the new year than with a development update. So, let's jump right in.
+
+We've spent a good amount of time fixing memory issues in the iRODS 4.3.0 server. All of that work is now merged and the server is more stable for it. We've also been working to improve thread-safety within the server.
+
+If you compile iRODS from source frequently, then you'll be happy to know that support for [Ccache](https://ccache.dev) has been added to the iRODS Development Environment. This enhancement has significantly improved compile-times. Iterating on the server has never been easier.
+
+We have an update about the GenQuery reimplementation. We've decided to ship it as a separate package. It will be maintained in its own repository so that we can get it in the hands of the community early. Doing that allows us to deploy bug fixes and enhancements frequently because it won't be tied to a particular version of the iRODS. We'll release more information as development moves forward. If that sounds interesting, then please get involved. We would love to hear feedback. You can follow along at [https://github.com/irods/irods_api_plugin_genquery2](https://github.com/irods/irods_api_plugin_genquery2).
```

Comment:
version of the iRODS -> "version of the iRODS server"

Reply:
Done.